eskip/eskip.go

```diff
@@ -17,8 +17,8 @@ import (
 const duplicateHeaderPredicateErrorFmt = "duplicate header predicate: %s"
 
 var (
-	duplicatePathTreePredicateError = errors.New("duplicate path tree predicate")
-	duplicateMethodPredicateError   = errors.New("duplicate method predicate")
+	errDuplicatePathTreePredicate = errors.New("duplicate path tree predicate")
```

Comment:
do we need to regenerate parser.y?

Reply:
Theses are not used in generated code but I'll double check if anything changes after regen.

Reply:
Its OK:
```console
$ go install golang.org/x/tools/cmd/goyacc@latest
$ go generate ./eskip
$ git diff
```

Reply:
Actually we can now remove `lint:file-ignore` directives from it.